public/Entity.js

```diff
@@ -50,8 +50,7 @@ class Entity {
     this.rect.width = Math.abs(1.5*this.xo*(this.hp = hp)/this.maxhp);
     if(this.xv < 0) this.sprite.scale.x = -1/3;
     if(this.xv > 0) this.sprite.scale.x = 1/3;
-    if(this.xv !== 0 || this.yv !== 0) this.requestedAnimation = data[this.type].action[1];
-    else this.requestedAnimation = data[this.type].action[0];
+    this.requestedAnimation = data[this.type].action[(this.xv !== 0 || this.vy !== 0) ? 1 : 0];
```

Comment:
this doesn't recreate the intended behavior... ?

Reply:
oh wait it does (`this.xv` and `this.`**`vy`**), its supposed to be (`this.xv` and `this.`**`yv`**)